libs/utils/loggers.py

```diff
@@ -0,0 +1,6 @@
+def get_source_log_directory(config):
```

Comment:
What about calling this file differently. Maybe logger_utils.py

Reply:
All the functions inside the utils folder don't include the `_utils`. If we change the name for this file we should also change the other ones.

Reply:
ok